src/extension.ts

```diff
@@ -127,7 +127,8 @@ export async function activate(context): Promise<extensionapi.ExtensionAPI> {
         registerCommand('extension.vsKubernetesLoad', loadKubernetes),
         registerCommand('extension.vsKubernetesGet', getKubernetes),
         registerCommand('extension.vsKubernetesRun', runKubernetes),
-        registerCommand('extension.vsKubernetesLogs', logsKubernetes),
+        registerCommand('extension.vsKubernetesShowLogs', logsKubernetes),
+        registerCommand('extension.vsKubernetesFollowLogs', (explorerNode: explorer.ResourceNode) => {logsKubernetes(explorerNode, true);}),
```

Comment:
I'm not keen on this style difference - it kind of disguises what the difference is between the two invocations (the unassuming second argument, rather than the big noisy ExplorerNode that appears three times in the text).  I'd suggest we either do both of them lambda style (with an explicit second argument in both cases), or create two functions (`showLogsKubernetes` and `followLogsKubernetes`) to be command handlers, both delegating to a core function.  (I prefer the latter but am okay with the former.)